src/tools/rustfmt/src/config/options.rs

```diff
@@ -142,13 +142,6 @@ pub enum HexLiteralCase {
     Lower,
 }
 
-#[config_type]
-pub enum ReportTactic {
-    Always,
-    Unnumbered,
-    Never,
-}
-
```

Comment:
Just want to double check that this change is necessary to land this PR. Since this isn't an import does it trigger the `unused_import` lint? If it's not necessary I'd prefer if this change was made with a PR to rustfmt.

Reply:
> Just want to double check that this change is necessary to land this PR

Yes, it's necessary unless `﻿allow[unused_imports]` is added here.

> Since this isn't an import does it trigger the unused_import lint?

I think this import statement is generted by `#[config_type]`: https://github.com/rust-lang/rust/blob/master/src/tools/rustfmt/config_proc_macro/src/item_enum.rs#L42

Reply:
Thanks for clarifying!